core/controllers/topic_editor.py

```diff
@@ -541,14 +544,29 @@ def get(self, topic_id: str) -> None:
         skill_creation_is_allowed = (
             role_services.ACTION_CREATE_NEW_SKILL in self.user.actions)
 
+        curriculum_admin_usernames = (
+            user_services.get_usernames_by_role('ADMIN'))
+
         self.values.update({
-            'classroom_url_fragment': classroom_url_fragment,
+            'classroom_url_fragment': (
+                None if (
+                    str(constants.CLASSROOM_URL_FRAGMENT_FOR_UNATTACHED_TOPICS)
```

Comment:
Why `str`? It's already a string, isn't it?

Reply:
yes it is but in the runtime the type is not know (it's any) as its extracted from a ts file.